queries/create-log.js

```diff
@@ -0,0 +1,32 @@
+/** IMPORTS */
+import { updateSudo as update } from '@lblod/mu-auth-sudo';
+import { sparqlEscapeString, sparqlEscapeUri, sparqlEscapeDateTime, uuid } from 'mu';
+
+/**
+ * TYPE: query
+ * Save error log to database
+ * Happends when a send a send mail does not arrive and returns an error
```

Comment:
Little typos : `Happends` --> `Happens` / a duplicated `a send`, should be `sent` also no ?

Reply:
ohh I didnt even notice that , will fix that realy quick :)